### ISSUE TITLE
Treasure Mob crash when in powered spawner.

### ISSUE THREAD

**Issue by Ranger1230** [grim3212/grim-pack#129]
###### Forge Version: 14.23.5.2768
------
I captured a treasure mob using an Ender IO soul vile and put it in a powered spawner. When the powered spawner tries to spawn the mob it gets a null pointer exception and crashes the client. here is the crash report. https://pastebin.com/ihGt27Qb

**Comment by grim3212** [comment]
Just tested it and wasn't able to reproduce.

Was this in Singleplayer of Multiplayer?
Were you doing anything special?
Does this happen every time?

When I tested I was using the creative spawner is this different to the one you were using besides the creative aspect?

I am not that familiar with Ender IO so any other information helps.

**Comment by Ranger1230** [comment]
Sorry it took me a while to reply.
I did not use creative for anything. I was able to reproduce it in a single player world.
The mods I had in the pack I reproduced it in where
Biomes O' Plenty: 7.0.1.2399
Ender IO: 5.0.37
EnderCore: 0.5.43
Grim Pack: 6.0.0.6
Just Enough Items: 4.13.1.22
Mekanism: 9.4.13.349

Steps,
I created a new world using the Biomes O' Plenty world gen (though I don't think that matters)
I used a spawn egg to spawn a Treasure Mob.
I gave myself an empty soul vile from Ender IO
I changed out of creative into survival
I used the soul vile to capture the treasure mob.
I changed back into survival.
I gave myself a broken spawner from Ender IO, a Soul Binder from Ender IO, a capacitor from Ender IO, and a fully charged energy cube from mechanism
I placed the energy cube and the soul binder next to each other (to power the soul binder)
I put the treasure mob soul vile and broken spawner in the soul binder and click the enchantment bottle button to start it.
Once it finished I took the broken spawner out of the soul binder.
I gave myself a vanilla anvil and a non creative Ender IO powered spawner, and another capacitor from Ender IO.
I combined the powered spawner and the broken spawner in the vanilla anvil.
I then placed the powered spawner next to the mechanism energy cube and then put the capacitor in it. A second or two after doing so my game crashed with that error.

Let me know if you still need more details.
